src/data/faq.json

```diff
@@ -352,19 +352,54 @@
+                        ]
+                    },
+                    {
+                        "title": "How do I use the combined 2G+ display?",
+                        "anchor": "admission_policy_2gplus_howto",
+                        "active": false,
+                        "textblock": [
+                            "If you have stored a valid vaccination or recovery certificate and a valid test certificate in the app, these will also be shown in a combined display from version 2.16 onwards. You can then use the switch to toggle between the QR code of the '2G certificate' (vaccination or recovery certificate) and that of the test certificate to quickly and easily provide 2G+ proof. Both QR codes must be checked on site using the CovPassCheck app together with presentation of the ID card if 2G+ proof is required. For privacy reasons, when the certificates are checked by the CovPassCheck app, it is not apparent whether the individual has a vaccination or recovery certificate. When scanning the QR codes, the checkers only see whether the certificate is valid or not.",
+                            "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_booster' target='_blank'>Will my booster vaccination be accounted for?</a></li><li><a href='#admission_policy_2gplus_not_shown' target='_blank'>Why is the 2G+ status proof not displayed even though I have a vaccination or recovery certificate and a negative test?</a></li></ul>"
+                        ]
+                    },
+                    {
+                        "title": "Why is the 2G+ status proof not displayed even though I have a vaccination or recovery certificate and a negative test?",
+                        "anchor": "admission_policy_2gplus_not_shown",
+                        "active": false,
+                        "textblock": [
+                            "The 2G+ status proof results, among other things, from the combination of vaccination, recovery <b>and</b> test certificates. If the data within the certificates, for example name or date of birth, do not match, the Corona-Warn-App cannot match these certificates and determine the corresponding status. In this case, the Corona-Warn-App will show you two certificates in the overview with 3G and 2G rules assigned to them respectively. Therefore, make sure that your name is entered correctly and fully when issuing the certificate, and contact the issuing office of the certificate in case of errors in the entry of the name or date of birth.",

```

Comment:
"when issuing the certificate" =>
"when the certificate is issued"

(since the user is not the person doing the issuing)

Reply:
Fixed in [3bdc4df](https://github.com/corona-warn-app/cwa-website/pull/2461/commits/3bdc4df6443f0ba0879d9564e3f1eac36b1f48d6), thanks!